outputs.tf

```diff
@@ -0,0 +1,8 @@
+output "webapp_url" {
+    value = "https://${azurerm_linux_web_app.main.default_hostname}"
+}
+
+output "web_hook" {
+    value = "https://${azurerm_linux_web_app.main.site_credential[0].name}:${azurerm_linux_web_app.main.site_credential[0].password}@${azurerm_linux_web_app.main.name}.scm.azurewebsites.net/docker/hook"
+    sensitive = false
```

Comment:
This should be `true` here, it has a password in it that might get accidentally logged to your actions currently

Reply:
corrected and set to true